buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -349,11 +349,24 @@ public short getShort(int index) {
     protected abstract short _getShort(int index);
 
     @Override
+    public short getShortLE(int index) {
+        checkIndex(index, 2);
+        return _getShortLE(index);
+    }
+
+    protected abstract short _getShortLE(int index);
```

Comment:
I guess this change can only go to 4.1 @normanmaurer? If we wanted it to go back to 4.0 we could just use the regular getters and just use the swap utility methods.